app/vendors/vendor-dashboard/testimonials/TestimonialsList.tsx

```
@@ -0,0 +1,103 @@
+
+export default function TestimonialsList({
+  testimonials,
+}: TestimonialsListProps) {
+  if (testimonials.length === 0) {
+    return (
+      <div className="flex flex-col items-center justify-center py-12 text-center">
+        <MessageSquare className="h-12 w-12 text-gray-300 mb-4" />
+        <h3 className="text-lg font-semibold mb-2">No testimonials yet</h3>
+        <p className="text-gray-500 mb-6 max-w-md">
+          Testimonials help build trust with potential clients. Request
+          testimonials from candidates you've worked with.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)